### ISSUE TITLE
Első issue

### ISSUE THREAD

**Issue by Kissgerg** [Uni-Sopron/cpp-o23-rpg#1]
Sose csináltam még ilyet. Első issue a bétára.

**Comment by hegyhati** [comment]
no, megneztem. Sokban epit az orai dolgokra ugye, de a Skill-es dolog eredeti otlet. Kodszervezes-ugyileg volna mit pofozni rajta tobb helyen, de ami a legfontosabb, hogy player es az enemy osztalyok nagy resze ugyanaz, azokat egy kozos ososztalyba kivenni illene. A kerdes az, hogy hanyast szeretnel erre kapni. 4-es megvan, ha a ososztalyt rendesen megcsinalod, akkor 5.
